install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -386,6 +386,15 @@ data:
   # https://github.com/cilium/hubble/blob/master/Documentation/metrics.md
   hubble-metrics: {{ .Values.global.hubble.metrics | join " " | quote }}
 {{- end }}
+{{- range .Values.global.hubble.listenAddresses }}
```

Comment:
While the intent of this section seems correct, this will not work as is. You cannot use command-line arguments in the config map.

Reply:
Not clear what you meant here.

Reply:
Still having some trouble adding this section, since it was constantly failing here. So i had to remove it. Any suggestions on what is best way to add it.

Reply:
So you copied over this section from a `Pod` description, but this is a `ConfigMap` object. You cannot use command-line arguments in the config map, you must use the key value format keys (which translate to environment variables).

Instead of `--listen-client-urls`, you must use the `hubble-listen-addresses`. Most of this code is already there on line 372 of this file.

I believe the reason why the code is not failing in CI right now is because the if-statement is always false. If it ever were true (i.e. if you switch to `.Values.global` as you eventually will have to), it would not work.